CMakePresets.json

```diff
@@ -42,7 +42,8 @@
             "inherits" : "default-release",
             "hidden": true,
             "cacheVariables": {
-                "CMAKE_INSTALL_PREFIX" : "/usr/local"
+                "CMAKE_INSTALL_PREFIX" : "/usr/local",
+                "MAKEFLAGS": "-j$(nproc)"
```

Comment:
not sure what this is for: we use ninja be default, it already utilizes all cores